code/modules/surgery/organs/tongue.dm

```diff
@@ -1,3 +1,5 @@
+#define TONGUE_MAX_HEALTH 560
```

Comment:
no offense but where are you pulling these numbers from because i see lungs max health being 350 and this is 560? organs really should have atleast somewhat similar health numbers imo.

Reply:
This was supposed to be 60, I didn't meant to hit the delete button so lightly before. The numbers are a complicated derivation considering the mass, young's modulus and size, tested across a sample size of at least 50 organs.

That is to say I made the numbers up roughly based off their irl size. Hopefully that's okay?